src/Raven.Server/Config/Categories/BackupConfiguration.cs

```diff
@@ -58,6 +58,10 @@ public class BackupConfiguration : ConfigurationCategory
         [ConfigurationEntry("Backup.CloudStorageOperationTimeoutInMin", ConfigurationEntryScope.ServerWideOrPerDatabase)]
         public TimeSetting CloudStorageOperationTimeout { get; set; }
 
+        [Description("Disable the direct upload to an external destination, instead use a temp file to create the backup first.")]
+        [DefaultValue(true)]
+        [ConfigurationEntry("Backup.DisableDirectUpload", ConfigurationEntryScope.ServerWideOrPerDatabase)]
+        public bool DisableDirectUpload { get; set; }
```

Comment:
I think I would rather have this as an option to a backup task and not global

Reply:
done